pitch.md

```diff
@@ -14,31 +23,170 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+From time to time, teachers and students of geometry and associated fields are
+interested in creating constructions. While sometimes they understand what they
+are creating, often times they are creating the idea as they make the drawing.
+
+It is also worth noting that while frequently users want these constructions
+rendered into drawings, the objects they are describing are geometric in
+nature - not inherently graphical. This means that sometimes geometrically
+distinct ideas may be spatially close. If input is spatial then this can result
+in geometric imprecision. For example, consider drawing a line from a triangle
+vertex to the midpoint of the opposing side vs bisecting that angle. Often
+these two actions are spatially similar (the lines end up close to each other).
+
+Current solutions to the problem fail to be both easy to use and geometrically
+precise. `tkz-euclid` is geometrically precise but very cryptic, and there are
+various visual construction engines which are easy to use but rely on spatial
+input and are thus geometrically imprecise in some cases.
+
+Furthermore, none of the above systems allow the user to see what steps they
+might take next.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Some parts of this solution are clearly a language. The way the user interacts
+with the system clearly must be a language. If they use some sort of textual
+system like
+
+```
+Line from A to B
+Line from A to C
+Bisect Angle C A B
+```
+
+then there is a language writen there with all of the nouns and verbs we
+expect.
+
+But there is also more. Under the hood, the system must have some sort of idea
+of how to represent these geometric ideas. A common form from which the
+description system, the rendering system, and the extrapolation system can all
+be build.
+
+I think that this language - this core representation of geometric ideas is the
+real DSL behind this project.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I've always been somewhat of a geometry enthusiast, which is probably the
+real origin of my interest in this project. I enjoy thinking about geometric
+ideas and think that I would have a lot of fun doing that for the next few
+weeks.
+
+That's not all though. I also generally have a lot of trouble using computers
+to foster creativity. I'm a very paper and pencil sort of guy. I think
```

Comment:
I feel like (at least for geometric constructions) I have the opposite stance - my freehand constructions (and constructions using a straight edge/compass take too long) are sufficiently inaccurate that I prefer using a computer to create mathematical figures.